### PR TITLE
N24: Add discussion links

### PR DESCRIPTION
**Pull request by 17cupsofcoffee**
_Part of #719_

### CODE REVIEW

File: content/news/024/index.md [hunk 1]
```diff
@@ -759,10 +759,12 @@ or [/r/rust_gamedev subreddit][/r/rust_gamedev] if you want to receive fresh new
...
-[/r/rust](TODO),
-[twitter](TODO).
 -->
+**Discussions of this post**:
+[/r/rust](https://www.reddit.com/r/rust/comments/p0hgjw/this_month_in_rust_gamedev_24_july_2021/),
+[/r/rust_gamedev](https://www.reddit.com/r/rust_gamedev/comments/p0hgsy/this_month_in_rust_gamedev_24_july_2021/),
+[Twitter](https://twitter.com/rust_gamedev/status/1424398304700420102).
+
 
 [/r/rust_gamedev]: https://reddit.com/r/rust_gamedev
```
Comment (17cupsofcoffee) [inline content/news/024/index.md:769]:
```suggestion
[Twitter](https://twitter.com/rust_gamedev/status/1424398304700420102).

[/r/rust_gamedev]: https://reddit.com/r/rust_gamedev
```

**Review by 17cupsofcoffee — COMMENTED** [review]
d'oh

**Review by ozkriff — COMMENTED** [review]
The template is a little bit outdated about this TODO. For the last few newsletter we've been sticking to the "Discuss this post on: /r/rust_gamedev, Twitter, Discord." format.

Also, one reddit link should be enough I guess?

**Comment by 17cupsofcoffee** [comment]
Fixed, and updated the template to match :+1:

**Review by ozkriff — APPROVED** [review]

**Pull request merged**